doc.css

```diff
@@ -1,63 +1,35 @@
-/*Header*/
-.top-nav {
-  display: flex;
-  flex-wrap: wrap;
-  min-width: 100%;
-  background-color: var(--secondary-color);
-  color: var(--sec-text);
-  padding: 0.8rem;
-}
-
-.nav-right {
-  margin-left: auto;
-}
-
-/* navigation */
-.navigation {
-  display: flex;
-  font-family: system-ui;
-}
-
-.navigation .nav-brand {
-  font-weight: bold;
-  font-size: 1.1rem;
-  color: var(--sec-text);
-}
-
-.navigation .link {
-  color: var(--sec-text);
-}
-
-.right {
-  text-align: right;
-}
-
-.navigation .link-active {
-  font-weight: bold;
-}
-
 /* hero*/
-
 .sec-hero {
-  display: flex;
+  display: grid;
+  grid-template-columns: 50% 50%;
   padding: 2rem;
   gap: 5px;
-  margin: auto;
+  margin: 5rem auto;
 }
 
 .sec-hero .hero-img {
-  max-width: 350px;
```

Comment:
convert it to rem

Reply:
thanks for the suggestion Prachi, I'll convert it to rem